### PR TITLE
Dependency updates - March 5 2024

### PR DESCRIPTION
**Pull request by Tristano8**

### CODE REVIEW

File: wai-handler-hal.cabal [hunk 1]
```diff
@@ -50,13 +50,13 @@ common deps
   build-depends:
     , base                  >=4.12      && <4.19
     , base64-bytestring     >=1.0.0.0   && <1.3
-    , bytestring            >=0.10.8    && <0.12.1
+    , bytestring            >=0.10.8    && <0.12.2
```
Comment (JackKelly-Bellroy) [inline wai-handler-hal.cabal:53]:
```suggestion
    , bytestring            >=0.10.8    && <0.13
```
If things are working with an `0.12.x` release, PVP says there won't be any breaking changes until `0.13`

File: wai-handler-hal.cabal [hunk 2]
```diff
@@ -50,13 +50,13 @@ common deps
...
     , case-insensitive      ^>=1.2.0.0
     , hal                   >=0.4.7     && <0.4.11 || >=1.0.0 && <1.2
     , http-media            ^>=0.8.1.1
     , http-types            ^>=0.12.3
     , network               >=2.8.0.0   && <3.2
-    , text                  ^>=1.2.3    || >=2.0   && <2.1.1
+    , text                  ^>=1.2.3    || >=2.0   && <2.1.2
```
Comment (JackKelly-Bellroy) [inline wai-handler-hal.cabal:59]:
```suggestion
    , text                  ^>=1.2.3    || >=2.0   && <2.2
```
For the same reason.

**Review by JackKelly-Bellroy — COMMENTED** [review]
Do you have the necessary Hackage permissions?

**Comment by Tristano8** [comment]
> Do you have the necessary Hackage permissions?

I do

**Review by JackKelly-Bellroy — APPROVED** [review]
Ace

**Pull request merged**